website/docs/reference/api-tokens-and-client-keys.mdx

```diff
@@ -21,6 +21,7 @@ This section describes what API tokens are. For information on how to create the
 Use API tokens to connect to the Unleash server API. API tokens come in four distinct types:
 
 -   [Admin tokens](#admin-tokens)
+-   [Project level access tokens](#project-access-tokens)
```

Comment:
It's not a different type of API token, so I don't think it should be listed here.

```suggestion
```